tests/cross/src/test/scala/tests/pc/CompletionSnippetSuite.scala

```diff
@@ -373,12 +366,9 @@ class CompletionSnippetSuite extends BaseCompletionSuite {
        |""".stripMargin,
     // additional completion when apply method is present
     compat = Map(
-      "3" ->
+      "3" -> // TODO: please double check
         """|Try
            |Try($0)
-           |TryBlock
-           |TryModule
-           |TryMethods
            |TryMethods
```

Comment:
Shouldn't it be as in Scala 2 ?
```suggestion
           |TryBlock
```

Reply:
Scala 2 is not filtering some things correctly. In this specific case, I remember `TryBlock` was either a `trait` or a `class`

Reply:
och makes sens. And TryMethods is a given, so looks correct.